addons/l10n_vn/data/account_tax_report_data.xml

```diff
@@ -144,64 +276,50 @@
                                 </field>
                             </record>
                             <record id="account_tax_report_line_03_01_02_vn" model="account.report.line">
-                                <field name="name">VAT on sales of goods and services 10%</field>
-                                <field name="code">VAT_ON_SALES_OF_GOODS_AND_SERVICES_10</field>
+                                <field name="name">VAT on sales of goods and services 8%</field>
+                                <field name="code">VAT_SALES_8</field>
```

Comment:
This makes the diff hard to read :smile: 
Can't you add the 8% line without changing the 10 into 8? It's ok if the XML id don't perfectly match (these kind of ids are annoying for this exact reason ^^)